### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

- Closes #301
- Closes #303
- Closes #302
- Closes #300
- Closes #299
- Closes #298

### CODE REVIEW

File: application-templates/starter/package.json [hunk 1]
```diff
@@ -19,9 +19,9 @@
     "@commercetools-frontend/ui-kit": "6.1.1",
     "prop-types": "15.6.2",
     "react": "16.7.0",
-    "react-apollo": "2.3.3",
+    "react-apollo": "2.4.0",
```
Comment (tdeekens) [inline application-templates/starter/package.json]: Added the ability to perform optimistic reads with readFragment.

File: application-templates/starter/package.json [hunk 2]
```diff
@@ -19,9 +19,9 @@
...
     "react-dom": "16.7.0",
-    "react-intl": "2.7.2",
+    "react-intl": "2.8.0",
```
Comment (tdeekens) [inline application-templates/starter/package.json]: Internal CLDR update.

File: packages/application-shell/package.json [hunk 3]
```diff
@@ -54,8 +54,8 @@
     "@flopflip/launchdarkly-adapter": "2.4.5",
     "@flopflip/memory-adapter": "1.1.0",
     "@flopflip/react-broadcast": "7.2.4",
-    "apollo-cache-inmemory": "1.3.12",
-    "apollo-client": "2.4.8",
+    "apollo-cache-inmemory": "1.4.2",
```
Comment (tdeekens) [inline packages/application-shell/package.json]: Quite big cache access improvements it seems according to changelog.

File: packages/sdk/package.json [hunk 4]
```diff
@@ -37,11 +37,11 @@
...
-    "@commercetools/api-request-builder": "4.1.1",
-    "@commercetools/sdk-client": "1.5.10",
-    "@commercetools/sdk-middleware-correlation-id": "1.0.1",
-    "@commercetools/sdk-middleware-http": "4.0.10",
-    "@commercetools/sdk-middleware-user-agent": "1.2.11",
+    "@commercetools/api-request-builder": "5.0.0",
+    "@commercetools/sdk-client": "2.0.0",
+    "@commercetools/sdk-middleware-correlation-id": "2.0.0",
+    "@commercetools/sdk-middleware-http": "5.0.0",
+    "@commercetools/sdk-middleware-user-agent": "2.0.0",
```
Comment (tdeekens) [inline packages/sdk/package.json]: This is all about dropping node@6 which doesn't matter at all for browsers.

**Comment by emmenko** [comment]
> chore: downgrade apollo-client

Uh, what happened here? Good that the starter app caught this error btw

**Pull request merged**